CODING.md

```diff
@@ -23,12 +23,13 @@
 It should be possible to follow this guide step by step to import a dataset from your personal test
 stack into nonprod.
 
+Production values are mentioned below.
+
 1. Set up properties of the target account, in this case nonprod:
 
    ```shell
-   export GEOSTORE_AWS_ACCOUNT_ID=632223577832
-   stack_prefix='nonprod'
-   export GEOSTORE_USER_ROLE_NAME="${stack_prefix}-api-users"
+   export GEOSTORE_AWS_ACCOUNT_ID=632223577832 # Use 715898075157 in production
```

Comment:
Shall we just reference the prod names for everything in the documentation? If someone wants to test something they'll probably just contact us.

Reply:
Yeah, agree.

Reply:
How about using the nonprod values in the coding documentation, since that's mostly for developers, and the production values in the usage documentation?

Reply:
Sounds good.